journalapp/views.py

```diff
@@ -1,24 +1,96 @@
+"""SQLAlchemy views to render learning journal."""
+from jinja2 import Markup
+import transaction
 from pyramid.response import Response
 from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound
 
 from sqlalchemy.exc import DBAPIError
 
 from .models import (
     DBSession,
-    MyModel,
-    )
+    Entry,
+)
+
+from journalapp.forms import EntryForm
+import markdown
+
+
+@view_config(route_name='list', renderer='templates/list.jinja2')
+def list_view(request):
+    """Return rendered list of entries for journal home page."""
+    try:
+        entries = DBSession.query(Entry).order_by(Entry.created.desc())
+        return {'entries': entries}
+    except DBAPIError:
+        return Response(CONN_ERR_MSG,
+                        content_type='text/plain',
+                        status_int=500)
```

Comment:
Let Pyramid handle returning a 500 error response.  No need to return this yourself.  Just let the database error raise.